homeassistant/components/integration/config_flow.py

```diff
@@ -34,56 +33,63 @@
 )
 
 UNIT_PREFIXES = [
-    {"value": "none", "label": "none"},
-    {"value": "k", "label": "k (kilo)"},
-    {"value": "M", "label": "M (mega)"},
-    {"value": "G", "label": "T (tera)"},
-    {"value": "T", "label": "P (peta)"},
```

Comment:
same issue here with the labels and values not matching previously

Reply:
That was wrong but on the PR G (Giga) is missing

Reply:
Opened https://github.com/home-assistant/core/pull/69109 - once that's merged I will rebase and fix here